src/Tizen.NUI/src/public/Common/NUIConstants.cs

```diff
@@ -136,48 +136,57 @@ public enum DrawModeType
     /// Enumeration for size negotiation resize policies.
     /// </summary>
     /// <since_tizen> 3 </since_tizen>
+    [Obsolete("Deprecated. Please use Tizen.NUI.BaseComponents.LayoutParamPolicies instead.")]
```

Comment:
Do you have a plan to add other type to LayoutParameterPolicies ?( UseNaturalSize, SizeRelativeToParent ... )

Reply:
For now, I don't have plan for that.
I think the current LayoutParamPolicies are sufficient for sizing usages.